content/03-reference/01-tools-and-interfaces/01-prisma-schema/index.mdx

```diff
@@ -93,12 +93,14 @@ model User {
 
 ## Using environment variables
 
-You can use environment variables to provide configuration options when a CLI command is invoked. This can e.g. be helpful in order to:
+You can use environment variables to provide configuration options when a CLI command is invoked. Environment variables allow you to:
 
 - Keep secrets out of the schema file
 - Improve portability of the schema file
 
-### Defining environment variables with the env function
+You can use machine-specific variables, `.env` files, or a combination of both. Refer to the `dotenv` documentation for information about [what happens if an environment variable is defined in two places](https://www.npmjs.com/package/dotenv#what-happens-to-environment-variables-that-were-already-set).
```

Comment:
I think the "Refer to ..." would better be positioned under "Using .env files" further down, and possibly the explanation really repeated here so people can pick it up from our own docs.